### PR TITLE
Update server setup script to use one central PRService only

### PR DESCRIPTION
**Pull request by zhenlan**
- Changed to setup PRService only if the id is "Master"
- Changed to grant app pool access to folders by inheritance instead of explicit-set
- Other cosmetic fixes

### CODE REVIEW

File: src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd [hunk 1]
```diff
@@ -50,104 +51,99 @@ if ERRORLEVEL 1 (
...
     if ERRORLEVEL 1 goto :Failure
 )
 
-:: Use current repo for cert gen if master repo does not exist
-if NOT EXIST %MasterRepo% (
-    set MasterRepo=%CurrentRepo%
-    set WcfServiceMaster=%WcfServiceName%
+:: Use current repo for cert generation if master repo does not exist
+if NOT EXIST %_masterRepo% (
+    set _masterRepo=%_currentRepo%
```
Comment (roncain) [inline src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd]: I know this is the same logic as last time, but it seems slightly dangerous to set _masterRepo when it isn't really the master repo.  There are other checks that %_masterRepo% != %_currentRepo%, so the pattern has been established they not be the same elsewhere.  Not blocking but would prefer creating a new env var like %_setupRepo% (or something like that) for the BuildCertUtil.cmd and CertificateGenerator.exe below.

**Comment by roncain** [comment]
LGTM.  One small optional nit recommendation.

File: src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd [hunk 2]
```diff
@@ -1,20 +1,21 @@
 @echo off
 setlocal
```
Comment (iamjasonp) [inline src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd]: I think you need elevated permissions to do all this, right? If so, consider detecting if the script is running elevated and bailing out if not with Usage. Suggestion only

File: src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd [hunk 3]
```diff
@@ -50,104 +51,99 @@ if ERRORLEVEL 1 (
...
 
 :: Create a new application pool and an application for the PR service
-echo Create IIS applicaton pool: %PRServiceName%
-call :Run %appcmd% add apppool /name:%PRServiceName% /managedRuntimeVersion:v4.0 /managedPipelineMode:Integrated
+echo Create IIS applicaton pool: %_prServiceName%
+call :Run %_appcmd% add apppool /name:%_prServiceName% /managedRuntimeVersion:v4.0 /managedPipelineMode:Integrated
 if ERRORLEVEL 1 goto :Failure
-echo Create IIS applicatoin: %PRServiceName%
-call :Run %appcmd% add app /site.name:"Default Web Site" /path:/%PRServiceName% /physicalPath:%CurrentRepo%\src\System.Private.ServiceModel\tools\PRService -applicationPool:%PRServiceName%
+echo Create IIS applicatoin: %_prServiceName%
```
Comment (iamjasonp) [inline src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd]: NIT: typo - "applicatoin"

File: src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd [hunk 4]
```diff
@@ -50,104 +51,99 @@ if ERRORLEVEL 1 (
...
 
 :: Setup PR service only if this is a master repo
-:: Uncomment line below when we switch to use one central PRService
-REM if /I NOT '%MasterRepo%'=='%CurrentRepo%' goto :SkipPRService
+if /I NOT '%_masterRepo%'=='%_currentRepo%' goto :SkipPRService
 
 :: Create a new application pool and an application for the PR service
-echo Create IIS applicaton pool: %PRServiceName%
-call :Run %appcmd% add apppool /name:%PRServiceName% /managedRuntimeVersion:v4.0 /managedPipelineMode:Integrated
+echo Create IIS applicaton pool: %_prServiceName%
```
Comment (iamjasonp) [inline src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd]: NIT: typo - "applicaton"

**Comment by iamjasonp** [comment]
A couple of optional NIT level comments/suggestions. LGTM

File: src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd (hunk 2 above)
Comment (zhenlan) [inline src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd]: fixed

File: src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd (hunk 3 above)
Comment (zhenlan) [inline src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd]: fixed

File: src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd (hunk 4 above)
Comment (zhenlan) [inline src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd]: fixed

File: src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd (hunk 1 above)
Comment (zhenlan) [inline src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd]: fixed

**Comment by roncain** [comment]
LGTM

**Pull request merged**